docs/docs/OAP-Developer-Guide.md

```diff
@@ -9,31 +9,15 @@ You can get more detailed information from OAP each module below.
 * [PMem Shuffle](https://github.com/oap-project/pmem-shuffle#5-install-dependencies-for-shuffle-remote-pmem-extension)
 * [Remote Shuffle](https://github.com/oap-project/remote-shuffle)
 * [OAP MLlib](https://github.com/oap-project/oap-mllib)
-* [Arrow Data Source](https://github.com/oap-project/arrow-data-source)
+* [Arrow Data Source](https://github.com/oap-project/native-sql-engine/tree/branch-1.1-spark-3.x/arrow-data-source)
 * [Native SQL Engine](https://github.com/oap-project/native-sql-engine)
 
 ## Building OAP
 
-### Prerequisites for Building
-
-OAP is built with [Apache Maven](http://maven.apache.org/) and Oracle Java 8, and mainly required tools to install on your cluster are listed below.
-
-- [Cmake](https://help.directadmin.com/item.php?id=494)
-- [GCC > 7](https://gcc.gnu.org/wiki/InstallingGCC)
-- [Memkind](https://github.com/memkind/memkind/tree/v1.10.1-rc2)
-- [Vmemcache](https://github.com/pmem/vmemcache)
-- [HPNL](https://github.com/Intel-bigdata/HPNL)
-- [PMDK](https://github.com/pmem/pmdk)  
-- [OneAPI](https://software.intel.com/content/www/us/en/develop/tools/oneapi.html)
-- [Arrow](https://github.com/Intel-bigdata/arrow)
-
-- **Requirements for Shuffle Remote PMem Extension**  
-If enable Shuffle Remote PMem extension with RDMA, you can refer to [PMem Shuffle](https://github.com/oap-project/pmem-shuffle) to configure and validate RDMA in advance.
-
-We provide scripts below to help automatically install dependencies above **except RDMA**, need change to **root** account, run:
+We provide scripts to help automatically install dependencies listed below **except RDMA**, need change to **root** user, run:
 
 ```
-# git clone -b <version> https://github.com/oap-project/oap-tools.git
+# git clone -b <tag-version> https://github.com/oap-project/oap-tools.git
```

Comment:
Add notes to explain branch-version.
# branch oap-1.1 corresponding to OAP 1.1
tag-version => branch-version